templates/onedev/index.ts

```diff
@@ -0,0 +1,58 @@
+import { Output, randomPassword, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const passwordPostgres = randomPassword();
+
+  services.push({
+    type: "postgres",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.databaseServiceName,
+      image: "postgres:15",
+      password: passwordPostgres,
+    },
+  });
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      source: {
+        type: "image",
+        image: input.appServiceImage,
+      },
+      proxy: { port: 6610, secure: true },
+      env: [
+        `HIBERNATE_DIALECT=io.onedev.server.persistence.PostgreSQLDialect`,
+        `HIBERNATE_CONNECTION_DRIVER_CLASS=org.postgresql.Driver`,
+        `HIBERNATE_CONNECTION_URL=jdbc:postgresql://${input.databaseServiceName}:5432/onedev`,

```

Comment:
I'm not sure if this will work. Let me know if you tested and worked properly.

In order to reach a database service, the host name is `[project]_[service]`, not just the service name.

Reply:
Thanks for catching that. I just updated the url and was able to deploy from schema without any issues.

Let me know if you find anything else that needs to be fixed 👍